src/main/java/gift/service/product/ProductService.java

```diff
@@ -16,28 +18,36 @@
 public class ProductService {
 
     private final ProductRepository productRepository;
+    private final CategoryRepository categoryRepository;
 
-    public ProductService(ProductRepository productRepository) {
+    public ProductService(ProductRepository productRepository,
+        CategoryRepository categoryRepository) {
```

Comment:
저는 보통 다음과 같이 개행합니다ㅎㅎ 그러면 읽기 좋더라구요! 취향 영역이라 굳이 안고쳐도 됩니다:)

```suggestion
        CategoryRepository categoryRepository
    ) {
```

Reply:
넵 감사합니다!